docs/MIR/mir-rust.md

```diff
@@ -19,7 +19,7 @@ C code for some system libraries that could be statically linked.
 ### Automation via `debian/rules`
 
 A good example for automating Rust vendoring is provided by
-[s390-tools](https://git.launchpad.net/ubuntu/+source/s390-tools/tree/debian/rules)
```

Comment:
Why would we want to change this one?
It is even wrong - Debian does not use the vendoring in the same way.

Thereby this can't be an example.
I know that LP git has trouble these days, but that change gets a Nack.
Once you dropped this you also no more need  MIR team ack as all else is only in other places.

Reply:
Thanks @cpaelzer. I wasn't sure about it, and I wanted to follow it up with a question but forgot to add that comment.

I changed it because LP doesn't seem to have code for `s390-tools`: https://launchpad.net/s390-tools

Reply:
> I changed it because LP doesn't seem to have code for s390-tools: https://launchpad.net/s390-tools

So, yeah, it's there; it's just me not understanding LP :/

However, the CGI there is so flaky it errors out all the time.

I'll revert this change and add the URL to ignore list.